content/en/docs/concepts/services-networking/ingress-controllers.md

```diff
@@ -56,6 +56,7 @@ Kubernetes as a project supports and maintains [AWS](https://github.com/kubernet
 * The [NGINX Ingress Controller for Kubernetes](https://www.nginx.com/products/nginx-ingress-controller/)
   works with the [NGINX](https://www.nginx.com/resources/glossary/nginx/) webserver (as a proxy).
 * The [ngrok Kubernetes Ingress Controller](https://github.com/ngrok/kubernetes-ingress-controller) is an open source controller for adding secure public access to your K8s services using the [ngrok platform](https://ngrok.com).
+* The [OCI Native Ingress Controller](https://github.com/oracle/oci-native-ingress-controller) is an Ingress controller for Oracle Cloud Infrastructure which allows you to manage the [OCI Load Balancer](https://docs.oracle.com/en-us/iaas/Content/Balance/home.htm).
```

Comment:
Based on the generated [deploy-preview-4153-services-networking/ingress-controllers](https://deploy-preview-41531--kubernetes-io-main-staging.netlify.app/docs/concepts/services-networking/ingress-controllers/#:~:text=ngrok%20platform.-,The,-OCI%20Native%20Ingress), changes LGTM.
 
 _Minor nit (Refer suggestion below)_
```suggestion
* The [OCI Native Ingress Controller](https://github.com/oracle/oci-native-ingress-controller#readme) is an Ingress controller for Oracle Cloud Infrastructure which allows you to manage the [OCI Load Balancer](https://docs.oracle.com/en-us/iaas/Content/Balance/home.htm).
```

Reply:
Done.